ProjectLighthouse.Tests.GameApiTests/Unit/Controllers/MessageControllerTests.cs

```diff
@@ -107,19 +107,6 @@ public async Task Announcement_WithEmptyString_ShouldBeEmpty()
         Assert.Equal(expected, announceMsg);
     }
 
-    [Fact]
-    public async Task Notification_ShouldReturn_Empty()
-    {
-        await using DatabaseContext dbMock = await MockHelper.GetTestDatabase();
-
-        MessageController messageController = new(dbMock);
-        messageController.SetupTestController();
-
-        IActionResult result = messageController.Notification();
-
-        Assert.IsType<OkResult>(result);
-    }
-
```

Comment:
Technically you should be writing new tests here for the new notification endpoint logic, but at this point, our test coverage is like 3% so it's not gonna make much of a difference